tests/RobotFramework/tests/cumulocity/self-update/tedge_self_update.robot

```diff
@@ -74,6 +73,42 @@ Refreshes mosquitto bridge configuration
     # Mosquitto should be restarted with new bridge
     Execute Command    cmd=sh -c '[ $(journalctl -u mosquitto | grep -c "Loading config file /etc/tedge/mosquitto-conf/c8y-bridge.conf") = 2 ]'
 
+Update tedge version from base to current using Cumulocity
+    Skip    msg="Until the dummy packages are included as test artefacts"
+
+    Execute Command    mkdir -p /setup/old-packages
+    Transfer To Device    /home/old-packages/*.deb    /setup/old-packages/
```

Comment:
Copying test debian packages from arbit locations like `/home/old-packages` is just temporary. These test packages will be checked-in along with the test via Git LFS or something.